crates/re_types/definitions/rerun/archetypes/series_line.fbs

```diff
@@ -38,4 +38,11 @@ table SeriesLine (
     ///
     /// Used in the legend.
     name: rerun.components.Name ("attr.rerun.component_optional", nullable, order: 3000);
+
+    /// Configures the zoom-dependent scalar aggregation.
+    ///
+    /// This is done only if steps on the X axis go below a single pixel,
+    /// i.e. a single pixel covers more than one tick worth of data. It can greatly improve performance
+    /// (and readability) in such situations as it prevents overdraw.
+    aggregator: rerun.components.AggregationPolicy ("attr.rerun.component_optional", nullable, order: 4000);
```

Comment:
Not a strong opinion but wonder if this should be `aggregation_policy` rather than `aggregator`.

Reply:
agreed, no reason here to stray away from the component name